pyproject.toml

```diff
@@ -0,0 +1,166 @@
+
+[tool.ruff]
+# Configuration: https://beta.ruff.rs/docs/configuration/
+# Rules: https://beta.ruff.rs/docs/rules/
+target-version = "py38"  # Pin Ruff to Python 3.8
```

Comment:
@thomas-bc I put Py3.8 minimum as the python 3.7 will be deprecated next months (no security support). So according to FPrime rules, Python 3.8 will be the minima version supported.